cmd/flowrunner/main.go

```diff
@@ -111,9 +111,9 @@ func RunFlow(assetsPath string, flowUUID assets.FlowUUID, initialMsg string, con
 
 	if initialMsg != "" {
 		msg := createMessage(contact, initialMsg)
-		repro.Trigger = triggers.NewMsgTrigger(env, flow.Reference(), contact, msg, nil, time.Now())
```

Comment:
I checked mailroom and this is always being set as now so.. goflow might as well set it itself like it does for resumes